LPFilter/Source/PluginProcessor.cpp

```diff
@@ -141,15 +151,31 @@ void LpfilterAudioProcessor::processBlock (AudioSampleBuffer& buffer, MidiBuffer
     // when they first compile a plugin, but obviously you don't need to keep
     // this code if your algorithm always overwrites all the output channels.
     for (int i = totalNumInputChannels; i < totalNumOutputChannels; ++i)
-        buffer.clear (i, 0, buffer.getNumSamples());
-
+        ioBuffer.clear (i, 0, ioBuffer.getNumSamples());
+    
+    // Define the block that passes into process function
+    dsp::AudioBlock<float> block (ioBuffer);
```

Comment:
Create a `process` method that implements the LPF processing using Juce DSP modules.
This will help you with the next implementations.